src/codex_plus/hooks.py

```
@@ -0,0 +1,964 @@
+                "gh", "pr", "view", "--json", "number,url", "-q", '"\(.number),\(.url)"',
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.DEVNULL
+            )
+            stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=0.3)
+            if stdout:
+                pr_data = stdout.decode().strip()
+                if "," in pr_data:
+                    pr_num, pr_url = pr_data.split(",", 1)
+                    pr_info = f"#{pr_num} {pr_url}"
+        except Exception:
+            pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)